client/client-multi/client-cpp/impl/utils/ThreadPool.cpp

```diff
@@ -114,32 +119,26 @@ void ThreadPool::shutdownNow()
 void ThreadPool::start()
 {
     for (std::size_t i = 0; i < workerCount_; ++i) {
-        workers_.push_back(std::thread(Worker(*this)));
+        workers_.emplace_back(Worker(*this));
```

Comment:
According to the official guide emplace_back() "constructs an element in-place at the end" (http://en.cppreference.com/w/cpp/container/list)